templates/index.html

```diff
@@ -70,40 +87,40 @@
     <h1 class="ui header">
       Tracking Demo
     </h1>
-
-    <p>Drag and drop or click to upload a video file for analysis.</p>
+    <ol>
+      <li>
+        Click <a href="/static/sample_video.mp4" download>here</a> to download a sample video.
+      </li>
+      <li>
+        Drag & drop the downloaded video into the rectangle below (or click on the rectangle to upload).
+      </li>
+      <li>
+        Wait until the video has been processed. Progress information will be posted to the screen.
+      </li>
+    </ol>	
 
     <div id="dropzone">
       <form class="dropzone needsclick" id="file-upload" action="/upload">
         <div class="dz-message needsclick">
           Drop files here or click to upload.
         </div>
       </form>
-    </div>
-
-    {% if jobs %}
+    </div>    
+    {% set session_is_empty = session|empty_session %}
```

Comment:
Can you please add a comment explaining what is going on here?

Reply:
https://github.com/MLDSAI/VisionTracing/pull/2/commits/64d01ee28c8807045a1a74d7ac8247063d9ba039